core/include/gnuradio-4.0/Block.hpp

```diff
@@ -362,10 +371,10 @@ class Block : public lifecycle::StateMachine<Derived>, protected std::tuple<Argu
     constexpr static TagPropagationPolicy tag_policy = TagPropagationPolicy::TPP_ALL_TO_ALL;
 
     //
-    using RatioValue = std::conditional_t<Resampling::kIsConst, const std::size_t, std::size_t>;
-    A<RatioValue, "numerator", Doc<"Top of resampling ratio (<1: Decimate, >1: Interpolate, =1: No change)">, Limits<1UZ, std::size_t(-1)>>      numerator   = Resampling::kNumerator;
-    A<RatioValue, "denominator", Doc<"Bottom of resampling ratio (<1: Decimate, >1: Interpolate, =1: No change)">, Limits<1UZ, std::size_t(-1)>> denominator = Resampling::kDenominator;
-    using StrideValue = std::conditional_t<StrideControl::kIsConst, const std::size_t, std::size_t>;
+    using RatioValue = std::conditional_t<Resampling::kIsConst, const std::uint32_t, std::uint32_t>;
+    A<RatioValue, "numerator", Doc<"Top of resampling ratio (<1: Decimate, >1: Interpolate, =1: No change)">, Limits<1UL, std::uint32_t(-1)>>      numerator   = Resampling::kNumerator;
+    A<RatioValue, "denominator", Doc<"Bottom of resampling ratio (<1: Decimate, >1: Interpolate, =1: No change)">, Limits<1UL, std::uint32_t(-1)>> denominator = Resampling::kDenominator;
```

Comment:
probably `std::numeric_limits<uint32_t>::max()` is more clear?

Reply:
replexed with:
```cpp
std::numeric_limits<RatioValue>::max()
```